activestorage/app/models/active_storage/variant.rb

```diff
@@ -50,7 +50,7 @@ def processed
 
   # Returns a combination key of the blob and the variation that together identifies a specific variant.
   def key
-    "variants/#{blob.key}/#{variation.key}"
+    "variants/#{blob.key}/#{Digest::MD5.base64digest(variation.key)}"
```

Comment:
Don't we need to require `digest` in this file? Or it is being required globally in activestorage?

Reply:
`Digest::MD5` appears to already be loaded here. It's also used in "/app/models/active_storage/blob.rb" without a `require`, not sure where it's getting loaded.